schema.sql

```diff
@@ -0,0 +1,10 @@
+/* Database schema to keep the structure of entire database. */
+
+CREATE TABLE animals(
```

Comment:
OPTIONAL 

Hi,

You have done a good job with creating the animals table with the required columns and data types. However, I have some suggestions for optional improvements that you can consider:

You can mark the id column as primary key to ensure that each row in the table has a unique identifier. This will help you to enforce data integrity and enable efficient queries and joins. You can do this by adding PRIMARY KEY at the end of the column definition or at the end of the table definition.